### PR TITLE
Fix native geolayer selector value conversion

### PR DESCRIPTION
**Pull request by jrmi**

### CODE REVIEW

File: src/modules/RA/Scene/components/GeolayerSelect.js [hunk 1]
```diff
@@ -69,8 +69,8 @@ const GeolayerSelect = ({ dataProvider, onChange, excludeIds = [], includeIds =
 
   const handleChoice = ({ target: { value } }) =>
     onChange({
-      geolayer: value,
-      label: geolayers.find(({ id }) => (value === id)).name,
+      geolayer: +value,
+      label: geolayers.find(({ id }) => (+value === id)).name,
     });
```
Comment (dtrucs) [inline src/modules/RA/Scene/components/GeolayerSelect.js:74]: If i'm right, this function is a select handle change and to retrieve the label you're building the same loop as you built the select tag. If this is the cas, you should have this information in the event target.

```
  const handleChoice = ({ target, target: { value, options } }) =>
    onChange({
      geolayer: +value,
      label: target[options.selectedIndex].textContent,
    });
```

File: src/modules/RA/Scene/components/GeolayerSelect.js [hunk 2]
```diff
@@ -69,8 +69,8 @@ const GeolayerSelect = ({ dataProvider, onChange, excludeIds = [], includeIds =
 
   const handleChoice = ({ target: { value } }) =>
     onChange({
-      geolayer: value,
-      label: geolayers.find(({ id }) => (value === id)).name,
+      geolayer: +value,
```
Comment (dtrucs) [inline src/modules/RA/Scene/components/GeolayerSelect.js:72]: I personally prefer the explicit `Number(value)` but it's the same

File: src/modules/RA/Scene/components/GeolayerSelect.js (hunk 1 above)
Comment (jrmi) [reply src/modules/RA/Scene/components/GeolayerSelect.js]: Not working with non native select.

File: src/modules/RA/Scene/components/GeolayerSelect.js (hunk 2 above)
Comment (jrmi) [reply src/modules/RA/Scene/components/GeolayerSelect.js]: Done.

**Pull request merged**